mesos-docker/run/run.sh

```diff
@@ -122,11 +122,37 @@ function check_if_container_is_up {
   fi
 }
 
+function quote_if_non_empty {
+
+  if [[ -n $1 ]];then
+    echo "\"$1\""
+  else
+    echo "$1"
+  fi
+
+}
+
+function check_mesos_version {
+
+  MIT_MESOS_HOST_VERSION=$(dpkg -s mesos | grep Version | awk '{print $2}')
+  MIT_MESOS_LIB_DOCKER_VERSION=$(docker exec $1 sh -c "dpkg -s mesos | grep Version" |  awk '{print $2}')\
+
+  if [[ ! "MIT_MESOS_HOST_VERSION" == "MIT_MESOS_LIB_DOCKER_VERSION" ]]; then
+    printMsg "WARN: Host and Docker images have different versions, Host:$MIT_MESOS_HOST_VERSION, Image: $MIT_MESOS_LIB_DOCKER_VERSION (image always reflects the latest version). Pls upgrade host."
+  fi
+}
+
+
 #start master
 function start_master {
 
   dip=$(docker_ip)
-  start_master_command="/usr/sbin/mesos-master --ip=$dip $MESOS_MASTER_CONFIG"
+
+  start_master_command="/usr/sbin/mesos-master --ip=$dip $(quote_if_non_empty $MESOS_MASTER_CONFIG)"
```

Comment:
`$MESOS_MASTER_CONFIG` has to be between `""`, otherwise only the first part (up to first space) of `MESOS_MASTER_CONFIG` will be returned.
What is what you want to avoid by forcing everything between quote, right?
Or you can use '$@' in `quote_if_non_empty`.